docs/03-extensions/magento2/index.mdx

```diff
@@ -24,3 +24,35 @@ First ensure you have installed the package:
 ```bash
 $ pnpm install @front-commerce/magento2@latest
 ```
+
+## Setup Magento2 Extension
+
+Update your `front-commerce.config.ts` to include the Magento2 Extension :
+
+```typescript title="front-commerce.config.ts"
```

Comment:
Thanks for adding this

**issue:** there are some missing configurations

**suggestion:** use the one from our demo / also documented in the "Get started > Installation" page

Reply:
added missing configurations